README.md

```diff
@@ -392,22 +392,46 @@ your-secretRootKey-value:
 
 [Back to top](#sap-business-technology-platform-sap-btp-service-operator-for-kubernetes)
 
-## Automatic Credentials Rotation
+## Automatic Service Binding Rotation
+
+You can rotate your service binding to ensure credentials are reset from time to time. Rotating means regularly generating new credentials while the old ones are discarded. 
+
```

Comment:
proposal: You can enhance security by rotating your service binding, ensuring that credentials are updated before they expire. Rotation involves generating new bindings with fresh credentials, while preserving and renaming the old ones for backup purposes until the transition is complete, after which they can be safely deleted.